pontoon/base/migrations/0134_google_translate_code.py

```diff
@@ -145,7 +145,7 @@ def populate_google_translate_code(apps, schema_editor):
         if pontoon_code in locale_map:
             locale_map[pontoon_code].google_translate_code = google_translate_code
 
-    bulk_update(locale_map.values(), update_fields=['google_translate_code'])
+    bulk_update(list(locale_map.values()), update_fields=['google_translate_code'])
```

Comment:
The same thing as in https://github.com/mozilla/pontoon/pull/1410/files#r332282796